x-pack/plugins/ml/public/application/jobs/new_job/pages/job_type/page.tsx

```diff
@@ -210,6 +211,22 @@ export const Page: FC = () => {
       }),
       id: 'mlJobTypeLinkrareJob',
     },
+    {
```

Comment:
Can you detect if the selected data view contains any geo fields and hide this option if there are no suitable fields?

Reply:
Added in 410b861be29ae50965d4075ab63a80e11f828488